Samples/InpaintHTTP/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+
+namespace InpaintHTTP
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Creating new server..");
+            WebHost webHost = new WebHost();
+            Thread.Sleep(-1);
```

Comment:
To avoid the main thread from finishing, i added a sleep that lasts `-1` to make the thread sleep for inf amount of time.
You might want to replace that line with the following code so you can terminate the application with a single key press after it launched the webhosting instance:
``Console.WriteLine("Press any key to exit...");
Console.ReadKey();``